app/serializers/api/user/public_user_serializer.rb

```diff
@@ -7,7 +7,17 @@ class PublicUserSerializer
 
       set_id :external_id
 
-      attributes :email, :chapter_ids, :stop_press_subscription
+      attributes :email, :chapter_ids, :stop_press_subscription, :my_commodities_subscription
```

Comment:
Does `:my_commodities_subscription` need to be added here as well if it's details are available in the subscriptions block?